### DISCUSSION TITLE
guestbook/

### DISCUSSION THREAD

**Discussion by giscus** [guangzhengli/blog-comments#4] [Announcements]
# guestbook/

Ladder@Guestbook - GuangzhengLi

https://guangzhengli.com/guestbook/

**Comment by wangyunzi** [comment]
从twitter过来的,博客很好看,简洁

**Comment by NameQuChen** [comment]
能加个好友位吗。想做的定制功能

**Reply by helloChenLei** [reply]
因为博主的主题我迁移了博客主题，同时做了一些修改。可以参考 https://demochen.com/posts/20230804/

**Comment by HotelEnglish** [comment]
感谢博主开源有趣的项目，我是一个大专院校的老师，准备带学生完成基于chatfiles的项目，在部署chatfiles后，由于服务器是国内的，所以运行后，后提示“获取模型时出错。请确保您的 OpenAI API 密钥已在侧边栏左下角设置。如果您已完成此步骤，OpenAI 可能遇到了问题。”这样的问题，根据网友在issue里的交流，在llm.py中添加了代理地址，类似：openai.api_base = "https://openai.aminer.cn/v1"这样的，换了很多个代理地址，包括自己搭建的可以正常使用的地址，都没能解决问题。特别想请教大佬给予指点，如何设置代理地址？感谢！

**Comment by AlainHsu** [comment]
请问怎么订阅博客呀？点击主页的 “RSS订阅” 是跳转到 https://guangzhengli.com/index.xml 乱码页。

**Reply by guangzhengli** [reply]
需要用 RSS 订阅相关的软件，比如 [RSSHub](https://docs.rsshub.app/)

**Comment by zhaozhentao** [comment]
感谢你的 k8s 教程

**Comment by jhhgo** [comment]
谢谢大佬的知识分享

**Comment by Juestin60** [comment]
请问博客的字体是什么，特别喜欢，以后我也想用这个字体写笔记，谢谢了

**Reply by guangzhengli** [reply]
[霞鹭文楷](https://github.com/lxgw/LxgwWenKai)

**Reply by Juestin60** [reply]
真心感谢！

**Comment by zsprain** [comment]
考虑写本书吗？

**Comment by markyun** [comment]
https://asciiart007.vercel.app/ 我的asciiart也用上了，感谢博主提供的主题

**Comment by ZimaBlueee** [comment]
博主, 你的博客字体怎么不见了...手机端没问题, pc端的自定义字体没了, 
我发现用该主题的站点都有同样的bug, 以前是没问题的

**Reply by ZimaBlueee** [reply]
现在自己又好了。。。之前字体一直在报错cdn有问题

**Reply by guangzhengli** [reply]
CDN 的问题，目前已经将字体的 CDN 源修改为 `cdn.jsdelivr.net`。

可以通过执行 `git submodule update --rebase --remote` 来获取更新

**Comment by shhspoilers** [comment]
谢谢分享，获益匪浅

**Comment by xiaofan-luan** [comment]
你好，来自Milvus社区，有空可以聊聊么？看到你的向量数据库文章，非常有意思
